README.md

```diff
@@ -39,22 +39,29 @@ recommendations are documented at [voxpupuli.org](https://voxpupuli.org/docs/rev
 ```puppet
 class foo (
   String $bar = '',
+  String[0] $foo = '',
+  Variant $baz = '',
```

Comment:
I'm still surprised this is legal. I've never seen code that actually has this. `Variant $baz` doesn't actually say anything. If it's accepted by Puppet, I think that deserves a bug report.

Reply:
yes. I raised https://tickets.puppetlabs.com/browse/PUP-11366